Documentation/PageTsconfig/Mod/WebList.rst

```diff
@@ -200,6 +200,62 @@ Example: Hide the column selector
 
     mod.web_list.displayColumnSelector = 0
 
+..  _pageTsConfigWebList-downloadPresets:
+
+downloadPresets
+===============
+
+..  confval:: downloadPresets.[table]
+    :name: mod-web-list-downloadPresets
+    :type: array of presets
+
+    ..  versionadded:: 13.2
+        A new feature has been introduced, adding the ability to pick the exported
+        data columns from a list of configurable presets.
```

Comment:
The `exported data columns` are the already exported columns, but I think you mean the data columns which has to be exported:

```suggestion
        A new function has been introduced that makes it possible to select the data columns
        to be exported from a list of configurable presets.
```